src/madpack/madpack.py

```diff
@@ -898,13 +900,19 @@ def run_install_check(args, testcase):
                   % (test_user, test_schema, schema)
 
         # Loop through all test SQL files for this module
-        sql_files = maddir_mod_sql + '/' + module + '/test/*.sql_in'
+        if is_install_check:
+            sql_files = maddir_mod_sql + '/' + module + '/test/*.ic.sql_in'
+        else:
+            sql_files = maddir_mod_sql + '/' + module + '/test/*.sql_in'
         for sqlfile in sorted(glob.glob(sql_files), reverse=True):
             algoname = os.path.basename(sqlfile).split('.')[0]
             # run only algo specified
             if (module in modset and modset[module] and
                     algoname not in modset[module]):
                 continue
+            # Do not run test/*.ic.sql_in files for dev-check.
```

Comment:
I believe if you use `*[!ic].sql_in` in the dev-check glob expression, it will exclude the ic files. You won't need these lines if that works.

Reply:
That's great, will try it out.